DistributionPackages/Neos.Demo.UserRegistration/Resources/Private/Fusion/Override/Form/Registration.fusion

```diff
@@ -1,38 +1,42 @@
 prototype(Neos.Demo:Content.Registration) {
-    process {
-        content.content.@process.appendHoneypot = afx`
-            {value}
-            <Neos.Fusion.Form:FieldContainer field.name="company" label=""
-                                             attributes.style="visibility:hidden;height:0;margin:0">
-                <Neos.Fusion.Form:Input attributes.class="p-0 text-sm"/>
-            </Neos.Fusion.Form:FieldContainer>
-        `
 
-        schema {
-            // Honeypot
-            company = ${Form.Schema.string().validator('StringLength', {minimum: 0, maximum: 0})}
-        }
-    }
+    renderer >
```

Comment:
Why does it need this? This would empty the renderer right?

Reply:
Yes. you always clear the renderer before you override it. Because there could be a danger that something from the original renderer could get into the overwriting. At least that's how I learned it. Maybe @markusguenther or @regniets can explain it a little better (or correct me if I'm wrong).

Reply:
IMHO this clears the renderer, but you don't set a new one. In this case we actually don't want to override the renderer, just something deeper on the path.

Reply:
Nevermind. I've tested it and it works in both cases.